layouts/financial.html

```diff
@@ -0,0 +1,10 @@
+<h1 id="financial">Financial</h1>
+
+<p>The Financial resources are used to support the costing, financial transactions and billing which occur within a healthcare setting, as well as payments which occur between healthcare providers and patients and insurers.</p>
+
+<% if @item[:version] == 'dstu2' %>
```

Comment:
We dont support a DSTU2 implementation of this so this can be removed.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#diff-fe1f2e7a3abf56af14947612b1d1d125L5

+1